resource-managers/kubernetes/core/src/main/scala/org/apache/spark/scheduler/cluster/kubernetes/KubernetesClusterSchedulerBackend.scala

```diff
@@ -322,7 +322,8 @@ private[spark] class KubernetesClusterSchedulerBackend(
             .addToRequests("cpu", executorCpuQuantity)
             .addToLimits("cpu", executorCpuQuantity)
           .endResources()
-          .withEnv(requiredEnv.asJava)
+          .addAllToEnv(requiredEnv.asJava)
+          .addToEnv(executorExtraClasspathEnv.toSeq: _*)
```

Comment:
Yeah unfortunately this slipped under the radar somehow. Are the SBT tests running now? Maybe SBT would have caught it and Maven doesn't?